tests/providers/google/cloud/hooks/test_mlengine.py

```diff
@@ -1009,11 +1009,11 @@ def test_delete_version(self, mock_get_conn, mock_project_id):
         model_name = "test-model"
         version_name = "test-version"
         # fmt: off
```

Comment:
This should be removed or moved. Same for other changes.

Reply:
Why should we remove it?

Reply:
Because… why should we keep it? The magic comment essentially turns Black off in the block, but if we are to replace single quotes anyway why not go all the way and enable Black?

Reply:
I think that the main reason to add these `fmt: off` is a conflict between black (which tries to makes everything in the same line) and E501

I will try to remove some of these fmt off/on